npm/package.json

```diff
@@ -11,6 +11,7 @@
 	},
 	"devDependencies": {
 		"rollup": "^0.50.0",
-		"rollup-plugin-replace": "^2.0.0"
+		"rollup-plugin-replace": "^2.0.0",
+		"webrtc-adapter": "^7.7.0"
```

Comment:
While 7.7.0 is indeed the latest version, the demos still only ship 7.4.0, as that's the latest you can find on cdnjs. Not sure if this is meant for server-side usage of janus.js, or for embedding in other frameworks, but maybe it would be best to stick to 7.4.0 here too? (unless this dependency downloads the right version independently of what the web page does).